Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -142,6 +173,228 @@ struct SelectTracks {
   }
 };
 
+//________________________________________________________________________________________________________________________
+
+/// Pre-selection of cascade secondary vertices
+/// It will produce in any case a HfTrackIndexProng2 object, but mixing a V0
+/// with a track, instead of 2 tracks
+
+/// to run: o2-analysis-weak-decay-indices --aod-file AO2D.root -b | o2-analysis-lambdakzerobuilder -b |
+///         o2-analysis-trackextension -b | o2-analysis-hf-track-index-skims-creator -b
+
+struct HFTrackIndexSkimsCreatorCascades {
```

Comment:
I would prefer to put it after `HFTrackIndexSkimsCreator`, not before.

Reply:
sure, it is before for no reason - I copied-pasted, and started from the copy at the top to be closer to the selection for practical reason.